extensions/azdata/src/localizedConstants.ts

```diff
@@ -4,33 +4,47 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as nls from 'vscode-nls';
+import { deploymentConfigurationKey, azdataAutoInstallKey, azdataAutoUpgradeKey } from './constants';
 const localize = nls.loadMessageBundle();
 
 export const searchingForAzdata = localize('azdata.searchingForAzdata', "Searching for existing azdata installation...");
 export const foundExistingAzdata = (path: string, version: string): string => localize('azdata.foundExistingAzdata', "Found existing azdata installation of version (v{0}) at path:{1}", version, path);
+export const notFoundExistingAzdata = localize('azdata.notFoundExistingAzdata', "Could not find existing azdata installation. Upgrade cannot be performed. Try installing instead");
```

Comment:
We should probably be asking them if they want to install in this case

Reply:
Yeah this constant won't be needed. It was created for case where user invoked azdata:upgrade command but there wasn't even an isntallation yet. With context aware commands this case won't arise, so I will be removing this constant.

Reply:
removed